NotebookAnalyzers/main.py

```diff
@@ -18,6 +18,14 @@
 }
 
 
+def _lint_notebooks_list(nb_objects):
```

Comment:
I think this function should be implemented as a method of the `Repository` class. As the `Notebook` class has its method `get_pynblint_results`, maybe the `Repository` class might have its own corresponding method (also called `get_pynblint_results`). 

The implementation is ok.